__tests__/spec/start.js

```diff
@@ -1,48 +0,0 @@
-/* eslint-env jest */
```

Comment:
What's the thinking behind deleting the tests in __tests__/spec/start.js? Is it because we need to rewrite the tests to get them to work, or is it because we don't want the functionality any more?

Reply:
`__tests__/spec/start.js` was only testing what happens if there's no `node_modules` directory but if there's no `node_modules` then there's no kit files to pick up that `node_modules` doesn't exist so it feels redundant.